code/src/index.css

```diff
@@ -11,3 +11,128 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+header {
+  background-color: #ffadad; 
+  border-bottom: 5px dotted;
+}
+header h1 {
+  font-family: "Courier New",
+    monospace;
+  margin: 0;
+  padding: 50px;
+  text-align: center;
+}
+footer {
+  background-color: #ffadad; 
+  border-top: 5px dotted;
+  font-family: "Courier New",
+    monospace;
+
+  padding: 30px;
+  text-align: center;
+}
+
+p {
+font-family: "Courier New",
+    monospace;
+    margin: 10px;
+}
+
+.newThought {
+  background-color: #f2f0f0;
+  box-shadow: 8px 8px black;
+}
+
+input {
+  width: 80%;
+  height: 30px;
+  margin: 8px;
+}
+.container {
+display:flex;
+flex-direction: column;
+min-width: 334px;
+max-width: 500px;
+margin: 0 auto;
+}
+
+.form-container {
+  margin: 10px;
+}
+.list {
+  width: 80%;
+  margin: 0 auto;
+  border:1px solid black;
+  margin-top: 30px;
+  box-shadow: 8px 8px black;
+}
+
+.grid {
+  display:grid;
+  Grid-template-columns: 1fr 1fr;
+}
+.message {
+  grid-column: span 2;
+}
+
+.loading-overlay {
+  position:absolute;
+  top: 0;
+  right: 0;
+  bottom:0;
+  left:0;
+  background: #c4c1c1;
+  display:flex;
+  justify-content: center;
+  align-items: center;
+}
+.loading-spinner{
+  width: 50px;
+  height:50px;
+  border: 2px solid #fa9696;
+  border-left: 2px solid #c4c1c1;
+  animation: spinner infinite 1.0s;
+}
+@keyframes spinner {
+    0% {
+      transform:rotate(0);
+    }
+
+    50% {
+      transform:rotate(180deg);
+    }
+    100% {
+      transform:rotate(360deg);
+    }
+  }
+
+
+
+
+
+
+
+.heart {
+  border-radius: 50%;
```

Comment:
Another solution is to just put a padding on for example 20 px on the heart. This will also make the circle round. It got to little space with the icon atm that is why it doesnt become round =)